CMakeLists.txt

```diff
@@ -301,7 +311,7 @@ endif()
 
 # setup the install rules
 if(NOT SFML_BUILD_FRAMEWORKS)
-    install(DIRECTORY include
+    install(DIRECTORY include "${PROJECT_BINARY_DIR}/configured/"
```

Comment:
Shouldn't this be `${PROJECT_BINARY_DIR}/configured/include/` instead to prevent potential future issues, where more than the include dir is configured?

Reply:
Only one thing is configured here, so it doesn't really matter. Do note that the trailing slash is meaningful, CMake uses rsync semantics for trailing slashes.

Reply:
is configured here... right now, but something could change and poof you gobble up all the headers. I rather be specific here to begin with, than later on remember to adjust it

Reply:
Installation code is touched by so few people that I think we can easily address problems as they come up instead of preemptively. If anything I'd prefer we collapse the `configured` directory and just put these headers right into `${PROJECT_BINARY_DIR}/include/` which would only require a very modest fix to our header installation rules but that's not required for me to approve this PR.

Reply:
I don't see how we gain anything from not being more specific. Since we filter by *.hpp and *.inl files and those should only ever exist in the include directory, there's no point in limiting the scope to begin with.

I also don't have anything against collapsing the configured directory, unless there's some technical reason why not to do so.